### ISSUE TITLE
Bug: new seeds get cleared before maturity by juicy queenbeet setup

### ISSUE THREAD

**Issue by benjaminysmall** [90259025/autosacrifice#3]
tl;dr: If the mod is targeting Juicy Queenbeet, it will plant a full 6x6 Queenbeet setup, overwriting any growing seeds that haven't yet been unlocked.

I _believe_ (but I'm not sure) that the last targeted seed before this was Drowsyfern. It might have been Wardlichen or something else, I wasn't watching super closely. Either way, the garden was set up like this (q = queenbeet, c = cronerice, d = drowsyfern, k = duketater)

| q | q | q | q | q | c |
|---|---|---|---|---|---|
| q |   | q |   | q | k |
| q | q | q | q | q |   |
| q |   | q |   | q |   |
| q | q | q | q | q | c |
|   |   | c |   |   | d |

Neither drowsyfern nor duketater had been unlocked yet; they were growing for the first time.

The mod said in the garden box at this point that it was "Targeting Juicy Queenbeet".

Once enough queenbeets had died, the mod attempted to replant the garden, and replanted it like this:

| q | q | q | q | q | q |
|---|---|---|---|---|---|
| q |   | q | q |   | q |
| q | q | q | q | q | q |
| q | q | q | q | q | q |
| q |   | q | q |   | q |
| q | q | q | q | q | q |

Which wiped away the progress toward Duketater and Drowsyfern.

I would have expected the mod to continue using the 5x5 setup until any new seeds along the edge had been unlocked, and only then, to start using the 6x6 setup.

Here's my save file, if it's helpful (although the mod is disabled at the moment):

Mi4wNDh8fDE2NjkwNzM5ODA4MTc7MTY2NDY0MTIzNTIzOTsxNjcwMTY2NTU5MDAwO1ZvbiBDaGlwO2l3aGxhfDAwMTExMDAxMTAxMDAwMTAwMTExMDEwMTAwMHw1LjYyOTk3ODg0MjUwOTcxNmUrNjM7NS42MzU3MDA1ODczNTg4MzdlKzYzOzY5ODczNjs0ODI4NTs1LjU3NTk0MzA1OTQzNTEwNWUrNjM7NTk2OzA7MDsyLjYyMTg1NDEyNTE2NDA1MmUrNjI7MDszMDY7NTIyMTU7MDstMTsyNDs4NTgwOzUuMzMxMjc3NTM2OTAzMzk1ZSs0Nzs4OTsxNDszNDQ5OzE4MjI1Mjg7MTc7Y2hyaXN0bWFzOzA7MDs2NDAwMzM2OTk4MTc1OTc3MDsxNDg4MjU4MDA1NzI5NjI1Mjs0OTEyMDc4OTkyNDQ2MzUyMDswOzA7Njk5Ozc2MDs4MDk7ODEwOzgxMTsyNjszOzE1OzA7MjA7MDswOzYzOzExOTsxNjcwMDk2MDU5OTEwOzA7MDsyMjcsMTI5OzEwMDsxOzE7MS4yMjg4MTcyNTkxMDEwNzQ3ZSs1MzswO3w5MTAsMTI2MCwxLjg5ODE1MzAyNzk1MDMzNjVlKzU5LDIsLDAsOTEwOzkwMCwxMjUwLDUuNDI5NjU1ODQ2NDM4NTcyZSs2MCwwLCwxLDkwMDs4NzAsMjA0MCw5LjI4ODYzOTU4MTQ3NzEyOWUrNTgsOSwxNjcwMTY2NjcwMjA3OjE6MTY3MDE2NTA4ODM1NTowOjEwMjc6NTg4MzoxOjM6MTY2OTA3Mzk4MDgzMDogMTExMTEwMTExMTExMTEwMDAwMDExMDAxMTEwMDAxMTEwMCAyMToxMToyMToxMToyMToxMDoyMToxMjoyMToxMjoyMToxNDoyMToxMjowOjA6MjE6MTA6MjE6MTM6MDowOjIxOjEyOjIxOjExOjIxOjExOjIxOjEzOjIxOjEyOjIxOjEzOjIxOjExOjIxOjExOjIxOjEzOjIxOjEyOjIxOjEwOjIxOjE0OjIxOjEyOjIxOjEzOjA6MDoyMToxMDoyMToxMzowOjA6MjE6MTQ6MjE6MTM6MjE6MTE6MjE6MTI6MjE6MTE6MjE6MTI6MjE6MTQ6LDAsODcwOzg2MCwyODcwLDIuMjY5Nzk0NDkwNjA5MDM5ZSs1OSwwLCwxLDg2MDs4NDAsMjgzMCwxLjY3NDU4ODQ2MjM3NDU3ODdlKzU4LDAsLDEsODQwOzgyMCwyODkwLDIuMTM1NDQ0NDIxNjQxNzFlKzU3LDEsMDowOjE6MDoxOiA3MDAwOjE6NTU6MjI0OjA6MDowOjAhNzQ4Nzo1OjE1MzozMTg6MDowOjA6MCEzMDMxOjU6MzY6MzQwOjA6MDowOjAhNzIwNzo1Oi01OjQ4MjowOjA6MDowITE1MTM2OjM6NDU6MjQ0OjA6MDowOjAhNzk3MTo0OjI4OjY1NzowOjA6MDowITEwNTk1OjQ6LTMxOjkyOjA6MDowOjAhMTAyMTY6MTo0MzozODE6MDowOjA6MCE0NTU2OjU6MzY6ODU6MDowOjA6MCE4OTk2OjA6LTg6NDg1OjA6MDowOjAhODQ4NTo1Oi02ODoxNjowOjA6MDowITU5NTM6MjotNTM6MTU0OjA6MDowOjAhMTczMzI6Mzo2NTo2MzE6MDowOjA6MCExNDkzODoxOjk6NjI4OjA6MDowOjAhMTQwMDU6NDoxMzo5MDowOjA6MDowITE0NTYyOjA6MDo1NDY6MDowOjA6MCExMDU1MTo0Oi05Nzo1Mjk6MDowOjA6MCEgMCwwLDgyMDs4MDAsMTE1MCwxLjc1MTQzOTUyMDA1Mzg4MjRlKzU5LDEsMTAvOC82IDMgMTY2OTE4MDUxODQ5MiAxLDAsODAwOzc4MCwyNTMxLDEuMDQ5ODEwNTk1NDMzNDE2MWUrNTcsMSwxMTcgNTYgNDYwIDEsMCw3ODA7NzYwLDE4NzAsMS44MjIxMzkxNTM3Mjg5ODZlKzU2LDAsLDEsNzYwOzc0MCwxODMwLDUuOTc3MzE2MDQ2MDAzMzY1ZSs1NiwwLCwxLDc0MDs3MzAsMTgxMCw2LjI5NDczOTQwMTE5Nzg5OWUrNTgsMCwsMSw3MzA7NzEwLDEwNjAsOC4wMTc3NTgxNTM3NzU2MDllKzU3LDAsLDEsNzEwOzY5MCwxMDQwLDUuMDc1NTU1MjU0NjcyMzdlKzU4LDAsLDEsNjkwOzY3MCwxMDIwLDEuNzA3NTc1NjI0NjMyOTkyZSs1OSwwLCwxLDY3MDs2NTAsMTAwMCwyLjEzNTk1Mjc2MTE4NDE4ODZlKzU5LDAsLDEsNjUwOzY0MCw5OTAsNy45OTQyODU0Nzk2MDEzNThlKzU5LDAsLDEsNjQwOzYwMCw5NTAsMS4wMjQzNTA5NzYyMTE1ODg4ZSs1OSwwLCwxLDYwMDs1NjAsOTEwLDMuNTExNjYyNzIxMDYyNTQyNmUrNTksMCwsMSw1NjA7NTUwLDkwMiwxLjY1NzM1NTc1NjIxNDk0OTRlKzYwLDAsLDEsNTUwO3wxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAwMTExMTExMDAxMDAwMTExMTExMTExMTAwMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAwMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAwMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMDAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMDEwMTAwMDExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAwMTAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMDAxMDExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMDAwMDExMTExMTExMTExMTExMTExMTExMTExMTExMTExMDExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAxMTExMTExMTExMTEwMDExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMDAwMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMDAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMDAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAxMTExMTExMTExMTAxMDEwMTAxMDEwMTF8MTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAxMTExMTExMDExMTExMTExMTExMTExMDAwMDAwMDExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMDExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAxMTExMTAxMTAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAwMDAwMDAwMDAwMDAwMTEwMDExMTExMTExMTExMDExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAxMTExMTExMTExMTExMTExMTExMDExMTExMTExMTEwMDAxMTAwMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMDExMTExMTExMTExMDExMTExMTExMTExMTExMTExMDExMTExMTExMTExMTExMTExMDEwMTEwMDExMTExMDAwMXwwLDUzNDAsMjYwOCw3O3xDQ1NFOnsidmVyc2lvbiI6IjIuMDMzIiwiQWNoaWV2ZW1lbnRzIjp7fSwiVXBncmFkZXMiOnt9LCJCdWlsZGluZ3MiOnt9LCJCdWZmcyI6e30sIlNlYXNvbnMiOnt9LCJPdGhlck1vZHMiOnt9LCJ2YXVsdCI6W10sInBlcm1hbmVudFVwZ3JhZGVzIjpbLTEsLTEsLTEsLTEsLTFdLCJjaGltZVR5cGUiOiJObyBzb3VuZCIsIm1pbGtUeXBlIjoiQXV0b21hdGljIiwiYmdUeXBlIjoiQXV0b21hdGljIn07Rm9ydHVuZUhlbHBlcjp7ImZvcnR1bmUiOnRydWUsImZvcnR1bmVhbGwiOnRydWUsImdvbGRlbiI6dHJ1ZSwiYWxzb3dyYXRoIjp0cnVlLCJyZWluZGVlciI6dHJ1ZSwid3JpbmtsZXIiOnRydWUsInJlc2VhcmNoIjp0cnVlLCJwbGVkZ2UiOnRydWUsImNsaWNrIjoxMCwiY2xpY2thbHdheXMiOmZhbHNlLCJmb3J0dW5lc291bmQiOmZhbHNlLCJnb2xkZW5zb3VuZCI6ZmFsc2UsIm11dGVjbGljayI6MCwiY2hvY29sYXRlZWdnIjp0cnVlfTtGb3J0dW5lIENvb2tpZTp7InNwZWxsRm9yZWNhc3RMZW5ndGgiOjIwLCJzaW1HQ3MiOjAsImNvbG9yT3ZlcnJpZGUiOnsiQnVpbGRpbmcgU3BlY2lhbCI6IiNGRjAwRkYiLCJDbGljayBGcmVuenkiOiIjNEJCOEYwIiwiRWxkZXIgRnJlbnp5IjoiI0UxQzY5OSIsIkZyZWUgU3VnYXIgTHVtcCI6IiNEQUE1NjAifSwiZm9yZWNhc3REcmFnb25Ecm9wIjp0cnVlfTtIb3J0aWNvb2tpZTp7ImFoTmV3IjoxLCJhaEp1aWN5IjoxLCJhaFVwZ3JhZGUiOjB9O2Nvb2tpZWdhcmRlbmhlbHBlcnJlbG9hZGVkOnsidGltZXJJbnRlcnZhbCI6MTAwMCwiYXV0b0hhcnZlc3QiOnRydWUsImF1dG9IYXJ2ZXN0TmV3U2VlZHMiOnRydWUsImF1dG9IYXJ2ZXN0TWF0dXJlZCI6ZmFsc2UsImF1dG9IYXJ2ZXN0QXZvaWRJbW1vcnRhbHMiOmZhbHNlLCJhdXRvSGFydmVzdFdlZWRzIjp0cnVlLCJhdXRvSGFydmVzdENsZWFuR2FyZGVuIjp0cnVlLCJhdXRvSGFydmVzdENoZWNrQ3BTTXVsdCI6ZmFsc2UsImF1dG9IYXJ2ZXN0TWluaUNwU011bHQiOnsidmFsdWUiOiI2LjUiLCJtaW4iOjB9LCJhdXRvSGFydmVzdER5aW5nIjpmYWxzZSwiYXV0b0hhcnZlc3REeWluZ1NlY29uZHMiOjUsImF1dG9IYXJ2ZXN0Q2hlY2tDcFNNdWx0RHlpbmciOmZhbHNlLCJhdXRvSGFydmVzdE1pbmlDcFNNdWx0RHlpbmciOnsidmFsdWUiOiIzNTAiLCJtaW4iOjB9LCJhdXRvUGxhbnQiOnRydWUsImF1dG9QbGFudEF2b2lkQnVmZnMiOmZhbHNlLCJhdXRvUGxhbnRSb3RhdGVTb2lsIjpmYWxzZSwiYXV0b1BsYW50Um90YXRlU29pbENvbWJvIjoxLCJhdXRvUGxhbnRDaGVja0NwU011bHQiOnRydWUsImF1dG9QbGFudE1heGlDcFNNdWx0Ijp7InZhbHVlIjoiNi41IiwibWluIjowfSwiYXV0b0ZvcmNlVGlja3MiOmZhbHNlLCJzYXZlZFBsb3QiOltbWzIxLDBdLFsyMSwwXSxbMjEsMF0sWzIxLDBdLFsyMSwwXSxbMjEsMF1dLFtbMjEsMF0sWzAsMF0sWzIxLDBdLFsyMSwwXSxbMCwwXSxbMjEsMF1dLFtbMjEsMF0sWzIxLDBdLFsyMSwwXSxbMjEsMF0sWzIxLDBdLFsyMSwwXV0sW1syMSwwXSxbMjEsMF0sWzIxLDBdLFsyMSwwXSxbMjEsMF0sWzIxLDBdXSxbWzIxLDBdLFswLDBdLFsyMSwwXSxbMjEsMF0sWzAsMF0sWzIxLDBdXSxbWzIxLDBdLFsyMSwwXSxbMjEsMF0sWzIxLDBdLFsyMSwwXSxbMjEsMF1dXSwicHJvdGVjdGVkU2VlZHMiOltdLCJhdXRvUGxhbnRSb3RhdGVTb2lsQ29tYm9OYW1lIjp0cnVlfTthdXRvc2FjcmlmaWNlOjIzNzVbUF0xNTc2W1BdMVtQXTFbUF0xW1BdMVtQXTE7TUVUQToqY29vbGVyIHNhbXBsZSBtb2QsKmxhbmcgc2FtcGxlIG1vZCwqc2FtcGxlIG1vZCxDQ1NFLEZvcnR1bmUgSGVscGVyLEZvcnR1bmUgQ29va2llLCpIb3J0aWNvb2tpZSxjb29raWVnYXJkZW5oZWxwZXJyZWxvYWRlZCwqYXV0b3NhY3JpZmljZTs%3D%21END%21
